sdk/debugger_interface/linux/gdb-sgx-plugin/gdb_sgx_plugin.py

```diff
@@ -686,6 +697,46 @@ def stop(self):
             gdb.execute(gdb_cmd, False, True)
         return False
 
+# This breakpoint is to handle enclave creation with buffer
+class CreateBufferEnclaveBreakpoint(gdb.Breakpoint):
+    def __init__(self):
+        gdb.Breakpoint.__init__ (self, spec="_create_enclave_from_buffer_ex", internal=1)
```

Comment:
As you mentioned in PR description, we rely on the existence of this symbol `_create_enclave_from_buffer_ex`. I think this may not be a problem because if the user is actually doing the debugging, s/he needs to install the libsgx-urts-dbgsym package anyway, and the symbols will be there.